pkg/cli/run.go

```diff
@@ -28,10 +29,14 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
+	humanOutputFormat = "human"
+)
+
 // Run executes terrascan in CLI mode
 func Run(iacType, iacVersion string, cloudType []string,
 	iacFilePath, iacDirPath, configFile string, policyPath []string,
-	format, remoteType, remoteURL string, configOnly, useColors bool) {
+	format, remoteType, remoteURL string, configOnly, useColors, verbose bool) {
```

Comment:
Hey Pankaj,

I misunderstood here--I was assuming that if we're now including the other details within the violation store, that there is no longer need for this verbose flag. Can we remove this?

Reply:
We would still need to have the verbose flag for the default human readable output.